physical/raft/fsm.go

```diff
@@ -784,7 +784,15 @@ func (f *FSM) Restore(r io.ReadCloser) error {
 
 	snapshotInstaller, ok := r.(*boltSnapshotInstaller)
 	if !ok {
-		return errors.New("expected snapshot installer object")
+		wrapper, ok := r.(raft.ReadCloserWrapper)
+		if !ok {
+			return fmt.Errorf("expected ReadCloserWrapper object, got: %~", r)
```

Comment:
What's `%~`? I've never seen that before.

Reply:
that's a bug :)